yt/frontends/boxlib/1

```diff
@@ -0,0 +1,86 @@
+class AMReXParticleHeader(object):
```

Comment:
Could you remove this file from the PR? I think all of this is already in `data_structures.py`